lib/api/src/js/module.rs

```diff
@@ -152,7 +186,8 @@ impl Module {
     /// Opposed to [`Module::new`], this function is not compatible with
     /// the WebAssembly text format (if the "wat" feature is enabled for
     /// this crate).
-    pub fn from_binary(_store: &impl AsStoreRef, binary: &[u8]) -> Result<Self, CompileError> {
+    pub fn from_binary(_store: &impl AsStoreRef, binary: impl IntoBytes) -> Result<Self, CompileError> {
+        let binary = binary.into_bytes();
         //
```

Comment:
and all the `IntoBytes` changes can also be consider API-breaking changes? (and so should go in 3.0?)

Reply:
Why we want to own the bytes @john-sharratt ?

Reply:
In order to pass it between threads - in particular it needs to be shipped to the other WebWorkers in the browser as they can not share the modules between each other, only the compiled bytes can be shared.